src/main/java/org/jabref/model/openoffice/util/OOTuple3.java

```diff
@@ -0,0 +1,13 @@
+package org.jabref.model.openoffice.util;
+
+public class OOTuple3<A, B, C> {
```

Comment:
Can you please comment on the usage of the class? (JavaDoc)

Reply:
I am not sure what do you mean here.
Inserted this:

```
/**
 * This class allows three objects to be packed together, and later
 * accessed as fields `a`, `b` and `c`.
 *
 * Can be used to avoid creating a new class for just this purpose.
 *
 * Can be useful if you do not have `Trifunction` at hand but need to
 * pass three objects at a time to a lambda function.
 *
 */
```

It is used in [RangeOverlapBetween.java](https://github.com/antalk2/jabref/blob/122d5133fa6c7b44245c5ba5600d398775718664/src/main/java/org/jabref/model/openoffice/rangesort/RangeOverlapBetween.java#L42)
to pack three values together without creating a class for that specific occasion.

Historically I created this class, then moved to ...Test when that was the only location
it was used, then eliminated its use, removed, then I needed it again. So here it is.
Not an absolute necessity, but I repeatedly needed it.